modules/querying/pages/func/query-user-defined-functions.adoc

```diff
@@ -171,87 +251,71 @@ PUT ExprUtil successfully.
 
 The `PUT` command will automatically store the files in all nodes in a cluster, overwriting any existing files that contain UDFs.
 
-Once the files are stored, you will be able to call the Query UDF the next time GSQL is executed. This includes the next time you start the GSQL shell or execute GSQL scripts from a bash shell. If you are using GraphStudio, however, you will be able to use the queries without needing to refresh the page.
+Once the files are stored, you will be able to call the Query UDF the next time GSQL is executed.
+This includes the next time you start the GSQL shell or execute GSQL scripts from a bash shell.
+If you are using GraphStudio, however, you will be able to use the queries without needing to refresh the page.
 
 .Example of a GSQL query that uses the UDF
-
 [source,gsql]
 ----
-CREATE QUERY udfExample() FOR GRAPH minimalNet {
+CREATE QUERY udf_example() FOR GRAPH Minimal_Net {
   DOUBLE x;
   BOOL y;
 
   x = 3.5;
   PRINT greater_than_three(x);
   y = greater_than_three(2.5);
   PRINT y;
-
 }
 ----
 
+After you are done uploading UDFs, it is strongly recommended that you disable uploading UDFs through GSQL until you need to modify the UDFs again:
+
```

Comment:
This part is not correct.
Once the `GSQL.UDF.EnablePutExpr` is set to false and takes effect, GSQL will load contents from `GSQL.UDF.ExprUtil` and `GSQL.UDF.ExprFunctions`. They are not necessarily the same as what user updated with PUT command.

If we want users to turn off `GSQL.UDF.EnablePutExpr` after users are satisfied with experimenting UDF file via PUT command. Then we need to ask them to manually set `GSQL.UDF.ExprUtil` and `GSQL.UDF.ExprFunctions` with the UDF files.

Without this step and just turning off PUT command will make GSQL to load user defined files from previous set `GSQL.UDF.ExprUtil` and `GSQL.UDF.ExprFunctions` values.

Reply:
Using PUT command update UDF files will not automatically change `gadmin config`'s `GSQL.UDF.ExprUtil` and `GSQL.UDF.ExprFunctions`

Reply:
Another notice about switching from PUT command to gadmin is that user should always set `GSQL.UDF.ExprUtil` and `GSQL.UDF.ExprFunctions` before or at the same time when the user sets `GSQL.UDF.EnablePutExpr` to false, apply config and restart gsql.

The case to AVOID is to set `GSQL.UDF.EnablePutExpr` to false, apply config and restart gsql, then set `GSQL.UDF.ExprUtil` and `GSQL.UDF.ExprFunctions`, apply config and restart gsql. The first restart of gsql will use previous set `GSQL.UDF.ExprUtil` and `GSQL.UDF.ExprFunctions` values which may not be what user wants to use.

Reply:
> I see. I only put this in because I saw this in the design files. I'll remove this paragraph.

I am okay with removing the paragraph here. But I do think it is worth to be mentioned somewhere in documentation what user needs to pay attention when they decide to switch between enable/disable PUT command.